deploy/test/test_server_config.rb

```diff
@@ -95,10 +95,16 @@
           })
 
         @s.bootstrap.must_equal true
+#        sleep(10)
         @s.validate_install.must_equal true
       end
     end
 
+    it "should deploy triggers" do
+      @s.deploy_triggers.must_equal true
+      @s.clean_triggers.must_equal true
+    end
+
```

Comment:
Is there a triggers-config.xml, or is the self-test running with the sample one?